src/test/java/org/apache/commons/configuration2/test/HsqlDB.java

```diff
@@ -35,7 +35,7 @@
 public class HsqlDB
 {
     private Connection connection = null;
-    private static Log log = LogFactory.getLog(HsqlDB.class);
+    private static final Log log = LogFactory.getLog(HsqlDB.class);
```

Comment:
Again, please, please, please, review ALL of your changes. If you make a static final, then the name should become UPPER CASE by convention.